### ISSUE TITLE
_id:"" in ionItem doesn't work

### ISSUE THREAD

**Issue by sclausen** [meteoric/meteor-ionic#149]
as far as I and the javascript compiler knows  is _id:"" invalid code
this drops an error:
```
{{#ionItem path="item.detail" _id:"" data="" query="" hash="" class=""}}...{{/ionItem}}
```
this doesn't, but also doesn't work
```
{{#ionItem path="item.detail" _id="" data="" query="" hash="" class=""}}...{{/ionItem}}
```
whatever I place in for _id is ignored, it always takes the current context.
unfortunately, I rely on mongo id's and I would like the hex representation of said id, not the function form like ObjectID("550169ede4b044639ac27261").

**Comment by linhkurt** [comment]
yeah, they say that this is iron-router issue. To by-pass this, make sure ionItem in the right context, otherwise, create a fake context for it, like:
You have <code>post={post_id, post_message, post_author, ....}</code>, 
and <code> router.map('/post/:post_id','post.detail'); </code>
Then you can do:
{{#with post}}
{{#ionItem path="post.detail"}}...{{/ionItem}}
{{/with}}
ionItem will automatically pickup post_id for you. It isn't necessarily named "_id", it can be whatever_id as long as you put ionItem in the right context with the right path.

**Comment by aguegu** [comment]
I have the same issue with @sclausen . It is quite annoying.

The problem is with the raw mongodb objectID object.
